### PR TITLE
update installation steps in documentation

### PR DESCRIPTION
**Pull request by sandeepgadhwal**
update installation steps in documentation

### CODE REVIEW

**Comment by sandeepgadhwal** [comment]
this fixes https://github.com/ArcGIS/geosaurus/issues/3193

File: guide/14-deep-learning/geospatial-deep-learning.ipynb [hunk 1]
```diff
@@ -55,16 +55,19 @@
     "\n",
```
Comment (rohitgeo) [inline guide/14-deep-learning/geospatial-deep-learning.ipynb]: <p>Add a markdown link to "Install deep learning dependencies..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

File: guide/14-deep-learning/object-detection.ipynb [hunk 2]
```diff
@@ -639,12 +639,8 @@
     "\n",
```
Comment (rohitgeo) [inline guide/14-deep-learning/object-detection.ipynb]: <p>Delete the phrase 'For&nbsp;<strong>arcgis.learn models</strong> please&nbsp;'. Start the sentence with "Refer to ..." and add a markdown link to the page getting rid of the URL in the rendered cell.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

**Comment by rohitgeo** [comment]
<p>@sandeepgadhwal can you please make the requested edits. Basically, start the sentence with "Refer to ..." and add a markdown link to the page getting rid of the URL in the rendered cell, like this:</p><p> </p><p>Refer to the [Install deep learning dependencies of arcgis.learn module] (<a href="https://developers.arcgis.com/python/guide/install-and-set-up/" target="_blank">https://developers.arcgis.com/python/guide/install-and-set-up/</a>) section&nbsp;for detailed documentation on installation of these dependencies.</p>

**Comment by rohitgeo** [comment]
cc: @DavidJVitale for inclusion in the developer website.

**Pull request merged**